### PR TITLE
workflows: Use GitHub actions from OpenEdu repo

### PR DESCRIPTION
**Pull request by teodutu**
The repository https://github.com/open-education-hub/actions houses multiple GitHub actions including `super-linter`, which was configured separately for the current repo. This PR changes the source of its linting actions to the aforementioned repo to unify the workflow among other repos under OpenEducationHub.

This PR also fixes an error caused by the default branch being previously set to `master` despite being changed to `main`: https://github.com/open-education-hub/open-education-hub-site/blob/bfdff1ec6138706ca9acf372abac81cf1f7ae2aa/.github/workflows/super-linter.yml#L26

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  spellcheck:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - name: Spellcheck
+        uses: open-education-hub/actions/spellcheck@main
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (gabrielmocanu) [inline .github/workflows/actions.yml]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```
The new line missing.

**Review by gabrielmocanu — CHANGES_REQUESTED** [review]

File: .github/workflows/actions.yml (hunk 1 above)
Comment (teodutu) [reply .github/workflows/actions.yml]: Sacré bleu!

**Review by gabrielmocanu — APPROVED** [review]

File: .github/workflows/actions.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: Linter Actions Workflow
+
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (gabrielmocanu) [inline .github/workflows/actions.yml]:
```suggestion
      - main
```

**Review by gabrielmocanu — CHANGES_REQUESTED** [review]

File: .github/workflows/actions.yml (hunk 2 above)
Comment (teodutu) [reply .github/workflows/actions.yml]: I was just typing to ask why the action isn't running...

**Comment by gabrielmocanu** [comment]
@teodutu 

Meanwhile, you can check this PR: https://github.com/open-education-hub/actions/pull/13
Otherwise, we will have the same problem.

**Comment by gabrielmocanu** [comment]
@teodutu 
I think `super-linter` works as intended rn.
What do you want to do about `spellcheck`?

**Review by gabrielmocanu — APPROVED** [review]

**Comment by teodutu** [comment]
@gabrielmocanu I'll try to fix the spellcheck errors before I merge this.

**Pull request merged**